app/schemas/models/selectors/cinematic.js

```diff
@@ -300,13 +300,25 @@ export const triggers = dialogNode => (dialogNode || {}).triggers
 
 const backgroundObject = triggers => {
   const bgObject = (triggers || {}).backgroundObject
-  if (!bgObject) {
+  if (!(bgObject || {}).thangType) {
     return
   }
 
   return _.merge(DEFAULT_THANGTYPE(), bgObject.thangType)
 }
 
+/**
+ * @param {Object} triggers
+ * @returns {number|undefined} number of ms before background object appears
+ */
+const backgroundObjectDelay = triggers => {
+  const bgObject = (triggers || {}).backgroundObject
+  if (!bgObject) {
+    return
+  }
+  return bgObject.triggerStart || 0
```

Comment:
Nitpick: `return ((bgObject || {}).triggerStart || 0)`, but this would return 0 if bgObject doesnt exist, instead of undefined. Feel free to ignore if the code calling it expects undefined in that case.

Reply:
Leaving it to keep the undefined return. But appreciate this comment a lot.